src/script/code-push-error.ts

```diff
@@ -6,8 +8,12 @@ export class CodePushError extends Error {
 }
 
 export class CodePushHttpError extends CodePushError {
-    constructor(message: string) {
-        super(message);
+    constructor(message: string, response: Http.Response) {
+        super(
+            message,
+            // @ts-expect-error - need to enable es2022 (see: https://github.com/microsoft/TypeScript/blob/main/src/lib/es2022.error.d.ts)
+            {cause: response}
```

Comment:
How does it work? There is no second parameter in `CodePushError` class.

Reply:
you right, I fixed